aitinero/src/app/home/layout.tsx

```diff
@@ -1,14 +1,17 @@
+import ConvexClientProvider from "../providers/ConvexClientProvider";
 import FormProvider from "../providers/FormProvider"
 
 export default function FormLayout({
     children,
-}: { 
-        children : React.ReactNode; 
+}: {
+    children: React.ReactNode;
 }) {
 
-    return(
+    return (
         <FormProvider>
-            {children}
+            <ConvexClientProvider>
+                {children}
+            </ConvexClientProvider>
```

Comment:
don't think this is necessary, `src/app/layout.tsx` has `<RootWrapper />` which contains `ConvexClientProvider`

Reply:
just saw it - have fixed :)